ngraph/core/include/ngraph/runtime/reference/interpolate.hpp

```diff
@@ -0,0 +1,509 @@
+//*****************************************************************************
```

Comment:
Please move your reference implementation for interpolate op to the [library with reference implementations](https://github.com/openvinotoolkit/openvino/tree/master/ngraph/core/reference).

Reply:
Done.